tests/foreman/cli/test_organization.py

```diff
@@ -1693,3 +1694,27 @@ def test_positive_info_by_name(self):
         org = make_org()
         result = Org.info({'name': org['name']})
         self.assertEqual(org['id'], result['id'])
+
+    @tier1
+    def test_positive_multibyte_latin1_org_names(self):
+        """Hammer Multibyte and Latin-1 Org names break list pagination
+
+        :id: 4fa0afe7-6d0a-4c3e-a0fc-4ecb95c50fc9
+
+        :BZ: 1418412
+
+        :expectedresults: Multibyte and latin1 names need to be
+            displayed with consistent spacing
+        """
+        org_names = valid_org_names_list()
```

Comment:
`valid_org_names_list()` is decorated with `@filtered_datapoint` which means it returns only 1 element if `run_one_datapoint` is set to `true` in `robottelo.properties`.
It may work ok for you locally (if `run_one_datapoint=false`), but `run_one_datapoint` is set to `true` on our CI, so instead of entire list you'll actually get only 1 random item (which may be something different from latin1/utf8).
In addition, according to test name and BZ description issue is not about alpha/alphanumeric names, so instead of reusing `valid_org_names_list()` which won't work anyway, consider forming your own list with cjk/latin1/utf8 names.